www-apps/redmine/redmine-3.4.12.ebuild

```diff
@@ -0,0 +1,232 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+USE_RUBY="ruby23 ruby24"
```

Comment:
ruby-2.3 has been removed from Gentoo, could you test this with 2.5 and 2.6 instead?

Reply:
Currently only ruby24 has all dependencies